Source/stores.cpp

```diff
@@ -1890,43 +1867,43 @@ void StorytellerIdentifyItem()
 	} else {
 		myPlayer.InvList[idx]._iIdentified = true;
 	}
-	myPlayer.HoldItem._iIdentified = true;
-	TakePlrsMoney(myPlayer.HoldItem._iIvalue);
+	item._iIdentified = true;

```

Comment:
This is only so that the item can be shown to the user in STORE_IDSHOW. Copying the identified item into StoreItem here would work, StoreItem isn't really relevant before this point.

Reply:
Sure, but doing so would require rewriting most of this function. Can we save that for an actual rewrite of the stores?